ui/index.html

```diff
@@ -1,12 +1,28 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <title>Hello World </title>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <link rel="stylesheet" href="style.css">
+
+    <!--Import Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!--Import materialize.css-->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css">
 </head>
 <body>
-    <input id="courseSearch" type="text" placeholder="Search Course">
-    <button type='submit' id="btn"> Submit</button>
-    <script type="text/javascript" src="main.js"></script>
+    <div class="autocomplete">
+        <input id='courseSearch' type="text" placeholder="Choose a course">
+        <div class="dialog">

```

Comment:
Can we rename this dialog to something like `search-dialog`? I think it's misleading because dialogs are mainly known as the alert dialogs. It's not too hard since it's just a replace all inside vs code